python/src/worker/result.py

```diff
@@ -16,18 +16,14 @@ def __init__(
         self.error = error
         self.cacheable = cacheable
 
-    def get_result_object(self, resp_format=False, s3_path=None):
+    def get_result_object(self, resp_format=False):
```

Comment:
do we actually use this `resp_format` for anything?

Reply:
yes, we use it, when `resp_format` is `True`, the result object is formatted to be uploaded to S3. When it is set to `False`, it is formatted to be sent to the UI (i.e. no inlining is done for the results)